### ISSUE TITLE
Cannot open settings

### ISSUE THREAD

**Issue by GSTDiandre** [mantou132/Spotify-Lyrics#126]
Right click doesn't seem to work, although middle click seems to work fine. Using Firefox.

**Comment by oRevived** [comment]
Same issue. Works fine on Chrome though

**Comment by mantou132** [comment]
You can go to the `about:addons` page for settings. I haven't figured out why there is an error in Firefox

**Comment by djwesterfield** [comment]
i'm a big noob, but is it because there isn't a js event for right click?

eg: 
```
function rightClick(clickEvent) { 
            clickEvent.preventDefault(); 
```

**Comment by GSTDiandre** [comment]
> You can go to the `about:addons` page for settings. I haven't figured out why there is an error in Firefox

Using this for now, but still unsure why a right click doesn't work. Works great otherwise.

**Comment by mantou132** [comment]
> i'm a big noob, but is it because there isn't a js event for right click?
> 
> eg:
> 
> ```
> function rightClick(clickEvent) { 
>             clickEvent.preventDefault(); 
> ```

No. I use a custom element to render the settings UI, but Firefox doesn't render this element correctly
